include/view.h

```diff
@@ -19,33 +19,33 @@
 namespace newsboat {
 
 class formaction;
```

Comment:
The project seems to work with the wrong forward declaration of formaction in there. I don't think view.h needs it. It might be a good idea to remove it at some point.

Reply:
You're right. I'll remove it after the merge.